community/graph-matching/src/test/java/matching/TestMatchingOfCircularPattern.java

```diff
@@ -224,6 +234,8 @@ public void verifyCount()
 
     private static Iterable<Node> traverse( final Node startNode )
     {
+        //        return new DynamicRelationshipType( name );
+        //        return new DynamicRelationshipType( name );
         return startNode.traverse( Order.BREADTH_FIRST, stopAtDepth( 2 ),
```

Comment:
Imagine the testing the JetBrains guys must do, to make this comment placement work reliably.